meta-signing-key/scripts/create-user-key-store.sh

```diff
@@ -310,5 +323,75 @@ create_user_keys() {
     create_rpm_user_key
 }
 
+if [ -d "$KEYS_DIR" ] ; then
+    echo "ERROR: $KEYS_DIR already exists, please remove it, to allow for the creation of new keys."
+    exit 1
+fi
+
+if [ ! -z "$GPG_KEYNAME" ]; then
+	gpg_key_name="$GPG_KEYNAME"
+else
+	echo -n "Enter GPG keyname [default: $gpg_key_name]: "
+	read val
+	if [ ! -z "$val" ] ; then
+		gpg_key_name=$val
+	fi
+fi
+
+if [ ! -z "$GPG_EMAIL" ]; then
+	gpg_email=$GPG_EMAIL
+else
+	echo -n "Enter GPG e-mail address [default: $gpg_email]: "
+	read val
+	if [ ! -z "$val" ] ; then
+		gpg_email=$val
+	fi
+fi
+
+if [ ! -z "$GPG_COMMENT" ]; then
+	gpg_email=$GPG_EMAIL
+else
+	echo -n "Enter GPG comment [default: $gpg_comment]: "
+	read val
+	if [ ! -z "$val" ] ; then
+		gpg_email=$val
+	fi
+fi
+if [ -z $GPG_PASS ]; then
+	while [ 1 ] ; do
+		echo -n "Enter RPM/OSTREE Passphrase: "
+		read val
+		if [ ! -z "$val" ] ; then
+			GPG_PASS=$val
+			break
+		fi
+	done
+fi
+if [ -z $IMA_PASS ]; then
+	while [ 1 ] ; do
+		echo -n "Enter IMA Passphrase: "
+		read val
+		if [ ! -z "$val" ] ; then
+			IMA_PASS=$val
+			break
+		fi
+	done
+fi
+
+
 create_user_keys
 
+cat<<EOF
+## The following variables need to be entered into your local.conf
+## in order to use the new signing keys:
+
+RPM_GPG_NAME = "$gpg_key_name"
+RPM_GPG_PASSPHRASE = "$GPG_PASS"
+RPM_FSK_PASSWORD = "$IMA_PASS"
+OSTREE_GPGID = "$gpg_key_name"
+OSTREE_GPG_PASSPHRASE = "$GPG_PASS"
+WR_KEYS_DIR = "$KEYS_DIR"
```

Comment:
WR_KEYS_DIR is misunderstanding a bit.

Reply:
There were some intermediate commits due to the fact this series came from a 3 other different branches.  You can see there is no WR_KEYS_DIR in the final version.   Do I need to squash the commits to create-user-key-store.sh ?